raspbian/applications/vcgencmd.md

```diff
@@ -0,0 +1,133 @@
+## vcgencmd
+
+`vcgencmd` is a command line utility that can get various pieces of information from the Videocore4 GPU on the Raspberry Pi. Much of the information available is only of use to internal Raspberry Pi engineers, but there are a number of very useful options available to end users that will be described here.
```

Comment:
Capital C in VideoCore. "internal" is superfluous in "internal Raspberry Pi engineers".